rpc/http.go

```diff
@@ -132,6 +139,9 @@ func (hc *httpConn) doRequest(ctx context.Context, msg interface{}) (io.ReadClos
 	if err != nil {
 		return nil, err
 	}
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		return resp.Body, fmt.Errorf(resp.Status)
```

Comment:
Oh that might be a classic format string vulnerability there. Shouldn't be `Errorf`.

Reply:
You can use `errors.New`.

Reply:
Thanks, fixed and also fixed travis failure (I hope)